Cartfile.resolved

```diff
@@ -9,4 +9,4 @@ github "luisobo/Nocilla" "0.11.0"
 github "wikimedia/NYTPhotoViewer" "88d2a1a7ac403439e310f55904b3768d257cccd9"
 github "wikimedia/Tweaks" "52db8948eaa85e0a52870237cf0ae2d50a55289a"
 github "wikimedia/hpple" "63951b4c33817321207bfa11ec850b22983cf03a"
-github "wikimedia/wikipedia-ios-codemirror" "77dce80e5da16b41ec2e2cd5e4feedb83297b29e"
+github "wikimedia/wikipedia-ios-codemirror" "24605fe5cab470928e915af5afe97c98f633a4f5"
```

Comment:
If you run `carthage update wikipedia-ios-codemirror` it'll update codemirror without re-building the other deps. Just for future reference - it's ok to leave the rebuilt deps on this PR